db/seeds/base/add_contracts.rb

```diff
@@ -0,0 +1,21 @@
+LeadProvider.find_each do |lead_provider|
+  Cohort.find_each do |cohort|
+    Course.find_each do |course|
+      if cohort.start_year < 2023 && (course.senco? || course.npqlpm?)
```

Comment:
A bit of a nit, maths is a special cours eand starts 2023, but isn't special in 2024. Senco starts 2024 and isn't a special course :)

Reply:
Updated 👍